### PR TITLE
Change tx CLI flag names from `a-chain`/`b-chain` to `src`/`dst`

### PR DESCRIPTION
**Pull request by seanchen1991**
Closes: #2410 

## Description

This PR changes the tx CLI command flags from using `a-chain`/`b-chain` terminology in favor of `src`/`dst` terminology in an effort to make the meaning behind the flags a bit more explicit, as well as align them with the language used in the help strings. 
______

### PR author checklist:
- [x] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [x] Linked to GitHub issue.
- [x] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [x] Reviewed `Files changed` in the GitHub PR explorer.
- [x] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: .changelog/unreleased/improvements/ibc-relayer-cli/2410-tx-cli-flags.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- Deprecate `a-chain`/`b-chain` tx CLI flag names in favor of `src`/`dst` flag
+  names ([#2410](https://github.com/informalsystems/ibc-rs/issues/2410)).
```
Comment (romac) [inline .changelog/unreleased/improvements/ibc-relayer-cli/2410-tx-cli-flags.md:2]:
```suggestion
- Rename `--a-` and `--b-` prefixes in `hermes tx` subcommands to `--src-` and `--dst-` 
  ([#2410](https://github.com/informalsystems/ibc-rs/issues/2410))
```

**Comment by ancazamfir** [comment]
Looks good. Are we doing the guide changes in a different (maybe release) PR?

File: relayer-cli/src/commands/tx/upgrade.rs [hunk 2]
```diff
@@ -323,11 +324,11 @@ mod tests {
     fn test_upgrade_chain_no_height_offset() {
         assert!(TxIbcUpgradeChainCmd::try_parse_from(&[
             "test",
-            "--receiver-chain",
+            "--reference-chain",
             "chain_receiver",
-            "--sender-chain",
+            "--sendhostin",
```
Comment (romac) [inline relayer-cli/src/commands/tx/upgrade.rs:329]: This should probably be:

```suggestion
            "--host-chain",
```

or is there a reason for this flag name here and in the other tests?

**Review by romac — CHANGES_REQUESTED** [review]
> Are we doing the guide changes in a different (maybe release) PR?

No we should do them here. We can now do guide updates in the same PR, no need to wait for the release PR.

File: relayer-cli/src/commands/tx/upgrade.rs (hunk 2 above)
Comment (romac) [reply relayer-cli/src/commands/tx/upgrade.rs]: ea56ae0fc84a16a2bf91c5f766141ffcffaca16b

**Review by romac — APPROVED** [review]

**Pull request merged**